### PR TITLE
Graduate node PIDS limiting to beta

### PR DESCRIPTION
**Pull request by RobertKrawitz**
Graduate node PID limiting to beta.

https://github.com/kubernetes/kubernetes/pull/76221/

### CODE REVIEW

**Comment by tengqm** [comment]
I'm not sure we should modify the release notes page directly.

**Comment by RobertKrawitz** [comment]
The only other change is to the list of feature gates supported by the kubelet; is that automatically generated or manually updated?  If it's automatically generated, then it sounds like there may actually not be any changes required here.

File: content/en/docs/reference/command-line-tools-reference/kubelet.md [hunk 1]
```diff
@@ -545,7 +545,7 @@ kubelet [flags]
...
...rue|false (ALPHA - default=false)<br/>LocalStorageCapacityIsolation=true|false (ALPHA - default=false)<br/>LocalStorageCapacityIsolationFSQuotaMonitoring=true|false (ALPHA - default=false)<br/>MountContainers=true|false (ALPHA - default=false)<br/>MountPropagation=true|false (ALPHA - default=false)<br/>PVCProtection=true|false (ALPHA - default=false)<br/>PersistentLocalVolumes=true|false (ALPHA - default=false)<br/>PodPriority=true|false (ALPHA - default=false)<br/>ReadOnlyAPIDataVolumes=true|false<br/>ResourceLimitsPriorityFunction=true|false (ALPHA - default=false)<br/>RotateKubeletClientCertificate=true|false (BETA - default=true)<br/>RotateKubeletServerCertificate=true|false (ALPHA - default=false)<br/>ServiceNodeExclusion=true|false (ALPHA - default=false)<br/>ServiceProxyAllowExternalIPs=true|false<br/>StreamingProxyRedirects=true|false (BETA - default=true)<br/>SupportIPVSProxyMode=true|false (ALPHA - default=false)<br/>SupportNodePidsLimit=true|false (BETA - default=true)<br/>TaintBasedEvictions=true|false (BETA - default=true)<br/>TaintNodesByCondition=true|false (BETA - default=true)<br/>VolumeScheduling=true|false (ALPHA - default=false)<br/>VolumeSubpath=true|false<br/>
```
Comment (kbhawkey) [inline content/en/docs/reference/command-line-tools-reference/kubelet.md]: hello @RobertKrawitz, kubelet.md is generated by docs at release time. It is my understanding that the content should have been updated at the source level. The release notes are published by a separate process. 
Is the `SupportNodePidsLimit` feature and feature state documented in, 
https://5cf69fdb73c51c000846bb31--kubernetes-io-vnext-staging.netlify.com/docs/reference/command-line-tools-reference/feature-gates/?

Comment (tengqm) [reply content/en/docs/reference/command-line-tools-reference/kubelet.md]: @kbhawkey, we can generate docs for all kube-xxx components except for kubelet. Kubelet broke the protocol for command line parsing. It only has naive special handling for command line help request and it is hiding all the rest from the doc generator. Therefore we can only manually maintain this doc. Sigh.  @mtaufen

Comment (kbhawkey) [reply content/en/docs/reference/command-line-tools-reference/kubelet.md]: That is odd. It appears to be auto-generated.

**Pull request merged**